adding new attribute for cloud front distribution resource (#517)
inspec/inspec-aws: docs/resources/aws_cloudfront_distribution.md

      it { should exist }
    end

## Parameters

### distribution\_id _(required)_

The ID of the CloudFront distribution. This parameter can be passed either as a string or as a `name: 'value'` key-value entry in a hash.

### disallowed\_ssl\_protocols _(optional)_

> If provided, this parameter is expected to be an array of strings identifying SSL/TLS protocols that you wish not to allow.

Reviewer: ```suggestion
If provided, this parameter is expected to be an array of strings identifying disallowed SSL/TLS protocol.
```

Author: fixed

|custom\_origin\_ssl\_protocols              | An array containing SSL/TLS protocols allowed by custom origins in this distribution, empty if there are no custom origins (i.e. only one or more standard S3 bucket origins). Current valid values are `SSLv3`, `TLSv1`, `TLSv1.1`, `TLSv1.2`. |
|custom\_origin\_protocol\_policy            | Custom origin protocol policy. It is a string, one of "http-only", "match-viewer", "https-only"
|viewer\_certificate\_minimum\_ssl\_protocol | The minimum SSL/TLS protocol version in the Viewer Certificate. Current valid values are `SSLv3`, `TLSv1`, `TLSv1_2016`, `TLSv1.1_2016`, `TLSv1.2_2018`, `TLSv1.2_2019`, `TLSv1.2_2021`. |
|s3\_origin\_config                          | True if there are any S3 origin configs in the distribution (i.e. standard S3 bucket origins), false otherwise. |

## Examples

### Test that a CloudFront distribution has secure protocols configured

    describe aws_cloudfront_distribution('DISTRIBUTION_ID') do
      its('VIEWER_CERTIFICATE_MINIMUM_SSL_PROTOCOL') { should_not match /SSLv3|TLSv1$|TLSv1_2016/ }

>       its('VIEWER_PROTOCOL_POLICIES') { should_not include 'ALLOW-ALL' }

Reviewer 2: @dkumaras Could you revert this change ? these are how actual implementation looks like and the upper case version of the attributes is misleading ..
